test/unit/rules/lint-no-input-block-test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+const message = require('../../../lib/rules/lint-no-input-block').message;
+
+generateRuleTests({
+  name: 'no-input-block',
+
+  config: true,
+
+  good: [
+    '{{button}}',
+    '{{#button}}{{/button}}',
```

Comment:
I’m not sure this is actually valid. Button would have to be a keyword? Can you update to x-button or something?

Reply:
👍

Reply:
Addressed